### PR TITLE
Modify concepts sections to point at Event-Centric future

### PR DESCRIPTION
**Pull request by JAG-UK**
[AB#10038](https://dev.azure.com/jitsuin/0629f48c-3979-4bbc-9026-cb06b3dfd0ae/_workitems/edit/10038)

### CODE REVIEW

**Comment by SteveLasker** [comment]
I'll pull locally to render and review today

**Review by SteveLasker — COMMENTED** [review]
Some nits, but great trimming.
~I'm getting hugo errors when attempting to build and render locally, so holding on the LGTM 'till I can confirm locally~
now building locally...

Pending acceptance/decline of the suggestions before LGTM, or any other updates.

File: content/platform/overview/core-concepts/index.md [hunk 1]
```diff
@@ -63,18 +74,20 @@ To make the whole history and individual events `UNEQUIVOCAL`, the root hash of
...
+Sharing the right amount of information with the consumers of your data is critical to creating a trustworthy shared history for any Asset. It is important that every participant be able to see and contribute to the Audit Trail without compromising security and personal private information. To ensure stakeholders can access only the information relevant to them, Events are private by default, unless posted to a [Public Asset](./#the-public-view). Tenant Administrators define how much of the Audit Trail a user or partner can see so that they only see what they need to complete a task.
+
+Like any high end transparency service, DataTrails operates a 'once see always seen' system, so while you remain completely in control of what Audit Trail data you share with your partners, it cannot be deleted or modified later.
```
Comment (jwimberley-dt) [inline content/platform/overview/core-concepts/index.md:79]: Should this be "once seen always seen"?

File: content/platform/administration/sharing-access-inside-your-tenant/index.md [hunk 2]
```diff
@@ -21,23 +21,24 @@ aliases:
...
 
-Attribute-Based Access Control (ABAC) policies can be used to control access to Assets, their attributes, and Events within a single organization.
+Attribute-Based Access Control (ABAC) policies can be used to control access Audit Trails and Events within a single organization.
 
 Specifically, ABAC policies are created by Administrators to share information with Non-Administrators in the same Tenancy.
 
 ABAC policies can be granular, with users only allowed to see single attributes at a time.
 
-It is possible to control policies based on types of Assets, their location, and whether Users can read or write any information in an Asset.
+It is possible to control read and write access to Trails based on a combination of properties of the Trail as specified in the Asset attributes, such as type and location (eg: "all models trained in Gondwalaland")
```
[the remark this answers is not in the document]
Comment (JAG-UK) [reply content/platform/administration/sharing-access-inside-your-tenant/index.md:30]: > Still references Asset. Should that be _Event attributes_, or still needs to be Asset 'till we retire Assets all together?

Unfortunately this needs to stay until we have the new system. It's the only way to define sharing policies.

File: content/platform/administration/sharing-access-inside-your-tenant/index.md [hunk 3]
```diff
@@ -21,23 +21,24 @@ aliases:
...
 
-By default, no Non-Administrators will see any existing Assets and Events unless an Administrator explicitly creates an ABAC policy to allow it.
+By default, no Non-Administrators will see any existing Trails or Events unless an Administrator explicitly creates an ABAC policy to allow it.
```
[the remark this answers is not in the document]
Comment (JAG-UK) [reply content/platform/administration/sharing-access-inside-your-tenant/index.md:32]:
```suggestion
By default, new users will not see any existing Trails or Events until an Administrator explicitly creates an ABAC policy to allow them.
```

File: content/platform/overview/core-concepts/index.md (hunk 1 above)
Comment (JAG-UK) [reply content/platform/overview/core-concepts/index.md:79]: Yes!

Comment (JAG-UK) [inline content/platform/overview/core-concepts/index.md:79]:
```suggestion
Like any high end transparency service, DataTrails operates a 'once seen always seen' system, so while you remain completely in control of what Audit Trail data you share with your partners, it cannot be deleted or modified later. 
```

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**